app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,40 @@
+<resources>
+    <string name="main_activity_description">Chat with your mentors.</string>
+
+    <string name="login">Login</string>
+    <string name="chat_with_mentor">Chat with a mentor</string>
+
+    <!-- NOTIFICATION SERVICE -->
+    <string name="notification_title">You have a new notification</string>
+
+    <!-- MAIN ACTIVITY -->
+    <string name="error_not_connected">You must first login!</string>
+    <string name="error_authentication_canceled">Login aborted</string>
+    <string name="error_no_internet">No internet connection!</string>
+    <string name="error_unknown_error">An unknown error occurred</string>
+    <string name="connection_succeed">Login successful!</string>
+    <string name="button_login_text_logged">View your profile</string>
+    <string name="button_login_text_not_logged">Login</string>
+
+    <!-- LOGIN ACTIVITY -->
+    <string name="toolbar_title_login_activity">Your Profile</string>
+    <string name="text_input_username">Username</string>
+    <string name="text_view_email">Email address:</string>
+    <string name="button_update_account">Update username</string>
```

Comment:
Change to "Change username", the FR phrase used does mean "update" but in this context it's definitely change. I don't know why French is like this sometimes.